pages/terra/appstream.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: Appstream
+description: "How to use Appstream Metainfo in Terra packages."
+---
+
+import { Callout } from "nextra/components";
+
+# Appstream Metainfo in Terra packages
+
+Terra supports Appstream metainfo, which enables your package to be picked up by graphical software 'store' applications, such as [KDE Plasma's Discover](https://apps.kde.org/discover/) or [GNOME's Software app](https://apps.gnome.org/Software/).
+This page will cover creating and installing appstream metainfo to your package, but we also highly encourage you to read over the documentation [here](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html).
+
+If you have difficulty populating the `.metainfo.xml` file, please seek support in our [Discord](https://fyralabs.com/discord) server, or make a draft pull request asking for assistance.
+
+<Callout type="info">
+  The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+  NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+  "OPTIONAL" in this document are to be interpreted as described in
+  [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119) and
+  [RFC 8174](https://www.rfc-editor.org/rfc/rfc8174).
+</Callout>
+
+## When Should I Use Appstream Metainfo?
+
+Packages satisfying any of the following cases are RECOMMENDED to provide the metainfo file.
```

Comment:
```suggestion
Packages that fit any of the following cases are RECOMMENDED to provide the metainfo file.
```
words flowing says this should say
"Packages that fit any of the following cases should provide a metainfo file."
but frankly i can't be assed to read the RFC RFC so i leave this an an exercise to the author